src/admin-views/aggregator/settings.php

```diff
@@ -174,7 +174,7 @@
 			'tooltip'         => esc_html__( 'Limit the number of imported events by number, date range, or not at all; on slower websites this may impact the success of imports. Selecting a shorter time period or a smaller number of events may improve results.', 'the-events-calendar' ),
 			'size'            => 'medium',
 			'validation_type' => 'options',
-			'default'         => 'range',
+			'default' => 'count',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 9 space(s) between "'default'" and double arrow, but found 1.

<sub>hash: 4f3bddbe80477b73b7d770289e3efa4f</sub>